build_tools/azure/install_pyodide.sh

```diff
@@ -14,9 +14,3 @@ pip install pyodide-build==$PYODIDE_VERSION pyodide-cli
 pyodide build
 
 ls -ltrh dist
-
-pyodide venv pyodide-venv
-source pyodide-venv/bin/activate
-
-pip install dist/*.whl
```

Comment:
Why isn't this step no longer required? Is it because we this is replaced by `npm install pyodide@$PYODIDE_VERSION`?

If so, why not call `npm install` as part of the `install_pyodide.sh` script?

Reply:
The Pyodide venv does not work with scipy for now (https://github.com/pyodide/pyodide/issues/3865), so you need to go the js wrapper way.

Reply:
Can we add a comment here or above `npm install pyodide@$PYODIDE_VERSION` to explain the situation?